DEA_datasets/DEA_Water_Observations.ipynb

```diff
@@ -0,0 +1,1409 @@
+{
```

Comment:
<p>"this is the principle _of_"? Doesn't really make sense as-is. "providing the individual water observations" doesn't really work without prior knowledge of WO; I suggest writing instead that it provides a classification of each pixel in each Landsat image as wet or dry. You do then define WOs in the next sentence, so you could also move that in front. I'd like Oxford commas here to make it a bit less ambiguous, but that's a personal preference...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
As discussed, totally agree but will aim to address this in CMI instead to keep that the point of truth